app/src/main/java/org/lineageos/tv/launcher/MainActivity.kt

```diff
@@ -103,6 +103,11 @@ class MainActivity : FragmentActivity(R.layout.activity_main) {
         registerReceiver(PackageReceiver(), intentFilter)
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        unregisterReceiver(PackageReceiver())
```

Comment:
You're unregistering a new PackageReceiver object, it won't unregister the registered one